torch_frame/config/image_embedder.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Callable
+
+from PIL import Image
+from torch import Tensor
+
+
+class ImageEmbedder(ABC):
+    r"""Parent class for the :obj:`image_embedder` of
+    :class:`ImageEmbedderConfig`. You should implement the
+    :meth:`forward_embed` which takes a list of images and returns
+    embeddings tensor. If you have your own retrieve function to
+    retrieve images from the paths, please override :meth:`forward_retrieve`
+    which takes the paths to images and return a list of
+    :obj:`PIL.Image.Image`.
+    """
```

Comment:
Before "You should implement the ...", can you write a high-level description of what it does, i.e., first retrieve images based on given paths stored in data frame and then embed the images into tensor.

Reply:
Also, "you should" -> "users are responsible for..." for softer wording